addons/website_hr_recruitment/controllers/main.py

```diff
@@ -30,17 +30,32 @@ def sitemap_jobs(env, rule, qs):
         '/jobs/department/<model("hr.department"):department>/office/<int:office_id>/employment_type/<int:contract_type_id>',
         '/jobs/country/<model("res.country"):country>/department/<model("hr.department"):department>/office/<int:office_id>/employment_type/<int:contract_type_id>',
     ], type='http', auth="public", website=True, sitemap=sitemap_jobs)
-    def jobs(self, country=None, department=None, office_id=None, contract_type_id=None, **kwargs):
+    def jobs_old(self, country=None, department=None, office_id=None, contract_type_id=None, **kwargs):
+        """ Keep old routes alive. """
+        return self.jobs(
+            country_id=country.id if country else None,
+            department_id=department.id if department else None,
+            office_id=office_id,
+            contract_type_id=contract_type_id,
+            **kwargs
+        )
+
+    @http.route([
+        '/jobs',
+        '/jobs/page/<int:page>',
+    ], type='http', auth="public", website=True, sitemap=sitemap_jobs)
+    def jobs(self, country_id=None, department_id=None, office_id=None, contract_type_id=None,
+             is_remote=False, is_other_department=False, is_untyped=None, page=1, search=None, **kwargs):
         env = request.env(context=dict(request.env.context, show_address=True, no_tag_br=True))
 
         Country = env['res.country']
         Jobs = env['hr.job']
+        Department = env['hr.department']
 
-        # List jobs available to current UID
-        domain = request.website.website_domain()
-        job_ids = Jobs.search(domain, order="is_published desc, sequence, no_of_recruitment desc").ids
-        # Browse jobs as superuser, because address is restricted
-        jobs = Jobs.sudo().browse(job_ids)
```

Comment:
Removing jobs but still used below... but not if I take the second commit... ok let's squash.